api-reference.rst

```diff
@@ -43,6 +43,31 @@ The aw_client package contains a programmer-friendly wrapper around the servers
    :members: ActivityWatchClient
    :undoc-members:
 
+aw_transform
+------------
+
+The aw_transform package contains transforms used in the query language.
+
+.. note:: Their function signatures and return types may deviate from how the transforms are actually implemented in the query language. For more details, see `aw_analysis.query2_functions`
+
+.. automodule:: aw_transform
+   :members:
+   :undoc-members:
+
+aw_analysis
```

Comment:
I guess that this name will be outdated pretty soon?

Reply:
Fixed